internal/fleet/agent_policy/resource_test.go

```diff
@@ -57,7 +57,7 @@ func TestAccResourceAgentPolicy(t *testing.T) {
 				ResourceName:            "elasticstack_fleet_agent_policy.test_policy",
 				ImportState:             true,
 				ImportStateVerify:       true,
-				ImportStateVerifyIgnore: []string{"skip_destroy"},
+				ImportStateVerifyIgnore: []string{"monitor_metrics", "monitor_logs", "skip_destroy"},
```

Comment:
What's happening here? This PR populates these fields on `Read`, why are we not setting them to computed?